libraries/core_libs/consensus/include/pbft/pbft_manager.hpp

```diff
@@ -587,7 +591,7 @@ class PbftManager : public std::enable_shared_from_this<PbftManager> {
   blk_hash_t soft_voted_block_for_this_round_ = NULL_BLOCK_HASH;
 
   // Full sync block for pbft block that is being currently cert voted for
-  SyncBlock cert_sync_block_;
+  PeriodData cert_sync_block_;
```

Comment:
based on the name of the variable and comment above, I am kind of lost what this variable actually holds ?

Reply:
Updated name and comment